src/Simplex/FileTransfer/Description.hs

```diff
@@ -204,17 +229,47 @@ validateFileDescription fd@FileDescription {size, chunks}
     chunksSize = fromIntegral . foldl' (\s FileChunk {chunkSize} -> s + unFileSize chunkSize) 0
 
 encodeFileDescription :: FileDescription p -> YAMLFileDescription
-encodeFileDescription FileDescription {party, size, digest, key, nonce, chunkSize, chunks} =
+encodeFileDescription FileDescription {party, size, digest, key, nonce, chunkSize, chunks, redirect} =
   YAMLFileDescription
     { party = toFileParty party,
       size = B.unpack $ strEncode size,
       digest,
       key,
       nonce,
       chunkSize = B.unpack $ strEncode chunkSize,
-      replicas = encodeFileReplicas chunkSize chunks
+      replicas = encodeFileReplicas chunkSize chunks,
+      redirect
     }
 
+newtype FileDescriptionURI = FileDescriptionURI Text
+  deriving (Eq, Show)
+
+instance StrEncoding FileDescriptionURI where
+  strEncode (FileDescriptionURI t) = encodeUtf8 t
+  strDecode = bimap show FileDescriptionURI . decodeUtf8'
+
+encodeFileDescriptionURI :: FileDescription 'FRecipient -> Maybe FileDescriptionURI
+encodeFileDescriptionURI fd = if T.length uri > qrSizeLimit then Nothing else Just (FileDescriptionURI uri)
+  where
+    uri = "simplex:/file#?d=" <> decodeUtf8 (urlEncode True yaml)
```

Comment:
use the same schema here as for invitations, to allow for simplex.chat and any other domain.